cmd/http/headers.go

```diff
@@ -80,6 +80,9 @@ const (
 	// Multipart parts count
 	AmzMpPartsCount = "x-amz-mp-parts-count"
 
+	// Indicate object expiration date/time
+	AmzExpiration = "X-Amz-Expiration"
```

Comment:
can we keep this lower case like `x-amz-expiration` and maintain the S3 header requirement just to be careful? I am doing that with `x-amz-version-id` as well